src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -159,7 +161,15 @@ public async Task<Recommendation> ApplyRecommendationPreviousSettings(Recommenda
             {
                 if (previousSettings.TryGetValue(optionSetting.Id, out var value))
                 {
-                    await  _optionSettingHandler.SetOptionSettingValue(recommendationCopy, optionSetting, value, skipValidation: true);
+                    try
+                    {
+                        await _optionSettingHandler.SetOptionSettingValue(recommendationCopy, optionSetting, value, skipValidation: true);
+                    }
+                    catch (UnsupportedOptionSettingType ex)
+                    {
+                        _interactiveService.LogInfoMessage($"Unable to retrieve value of '{optionSetting.Name}' from previous deployment. Make sure to set it again prior to redeployment.");
```

Comment:
Change `LogInfoMessage` to `LogErrorMessage`.

Reply:
Fixed